README.md

```diff
@@ -0,0 +1,52 @@
+MIES is a sweep based data acquisition software package for intracellular
+electrophysiology (patch clamp). It offers top of its class flexibility and
+robustness for stimulus generation, data acquisition/evaluation and has
+proven to be a valuable tool in the literature \[[1](https://doi.org/10.1038/s41593-019-0417-0),
+[2](https://elifesciences.org/articles/37349)\].
+
+## Highlights
+- Acquire data on up to eight headstage using up to five DACs simultaneously
+- Create arbitrarily complex stimulus sets with an easy to use GUI
+- Export **all** data including **all** of its metadata into the industry-standard
+  [NWBv2-format](https://alleninstitute.github.io/MIES/IPNWB/index.html) and
+  read it back in
+- Run custom code during data acquisition for [Automatic Experiment
+  Control](https://alleninstitute.github.io/MIES/file/_m_i_e_s___analysis_functions_8ipf.html#file-mies-analysisfunctions-ipf)
+- Completely automate the experimental setup using [JSON configuration files](https://alleninstitute.github.io/MIES/file/_m_i_e_s___configuration_8ipf.html#file-mies-configuration-ipf)
+- Interact with MIES from other programming languages (Python, C++, Javascript, ...) using [ZeroMQ](https://alleninstitute.github.io/MIES/ZeroMQ-XOP-Readme.html).
+- Use the integrated [scripting
+  language](https://alleninstitute.github.io/MIES/SweepFormula.html#the-sweep-formula-module)
+  for convenient on-the-fly data evaluation
+- Fully backwards compatible with **every** earlier MIES version
```

Comment:
- Acquire data on up to eight headstage on a single DAC.
- Run up to five DACs in parallel.
- Create arbitrarily complex stimulus sets with an easy to use GUI
- Export all data, including all of its metadata, into the industry-standard NWBv2-format and read it back in
- Run custom code during data acquisition for Automatic Experiment Control
- Completely automate the experimental setup using JSON configuration files
- Interact with MIES from other programming languages (Python, C++, Javascript, ...) using ZeroMQ.
- Use the integrated scripting language for convenient on-the-fly data evaluation.
- Fully backward compatible with every earlier MIES version

Reply:
@t-b we should mention something like:
- comprehensive experimental metadata acquisition and browsing tools